attributes/default.rb

```diff
@@ -1,7 +1,7 @@
-default['pgd']['pgd_path'] = "/home/vagrant/pgd"
-default['pgd']['virtualenv_path'] = "/home/vagrant/vrt"
-default['pgd']['user'] = "vagrant"
-default['pgd']['group'] = "vagrant"
+default['pgd']['pgd_path'] = "/home/centos/pgd"
```

Comment:
as a replacement, the attributes in `default.rb` should probably be better defaults, and the vagrant-specific ones should go in the `.kitchen.yml`.